app/policies/employees/user_policy.rb

```diff
@@ -28,5 +28,13 @@ def destroy?
     def upload?
       manager?
     end
+
+    def lock?
```

Comment:
such things could be done via alias, so you have one method doing work, and others are just aliases for this method